server/api/offline_results.py

```diff
@@ -64,12 +64,20 @@ def validate_offline_results(
         .filter_by(jurisdiction_id=jurisdiction.id)
         .count()
     )
+
     for contest in jurisdiction.contests:
         num_ballots = (
             ballot_draws_by_contest.get(contest.id, 0)
             if contest.is_targeted
             else ballots_sampled
         )
+
+        # Special case: if we sampled all ballots, then the max results allowed
+        # should be based on the ballot manifest, since we don't have any
+        # sampled ballots in the db
+        if sampled_all_ballots(round, election):
+            num_ballots = jurisdiction.manifest_num_ballots
```

Comment:
nit: overwriting the variable just computed, maybe cleaner to have an if/else statement here for this case and the case just prior.